File: src/main/java/com/liao/Hello.java
package com.liao;

public class Hello {
    public static void main(String[] args){
//        System.out.println("Hello World");
//        new Person().hello();
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));

        Person p = new Person("Liao",66.5f, 1.7f);
        p.hello();
        System.out.println(p.bmi());
        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean enroll = false;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/liao/Hello.kt
package com.liao

fun main() {
//    println("Hello Kotlin")
//    Human().hello()
    val h = Human(weight = 66.5f,height = 1.7f)
    println()
    /*var age = 19
    age =20
    var weight = 66.5f
    var name : String
    name = "Liao"*/
}

class Human(var name : String = "", var weight : Float, var height : Float){
    init {
        println("test $weight")
    }
//    constructor(name: String, weight: Float, height: Float) : this(weight, height)
    fun bmi() : Float{
        val bmi = weight / (height*height)
        return bmi
    }
    fun hello(){
        println("Hello Kotlin")
    }

}